airdrop/README.md

```diff
@@ -51,22 +51,13 @@ In order to run the tests you will need to build both the simple-asset and airdr
 
 First, make sure that you are in the root of the simple-asset project i.e. `/path/to/airdrop/simple-asset/<you are here>`.
 
-There is one command required to run the tests for simple-asset. This will both build and test the simple-asset project.
+There are two commands required to run the tests for simple-asset. This will build and test the simple-asset project.

```

Comment:
```suggestion
There are two commands required to run the tests for the simple-asset. The first command builds the contracts and the second command runs the tests.
```

Reply:
Note that the sentence above this describes the simple asset however that is not where all the tests are. The context above needs to be updated to make sure that all the tests are run.